modules/core/02-client/keeper/keeper.go

```diff
@@ -383,6 +383,28 @@ func (k Keeper) GetAllClients(ctx sdk.Context) (states []exported.ClientState) {
 	return states
 }
 
+// GetAllClientIDsOfType will iterate over the provided client type prefix in the client store
+// and return a list of clientIDs associated with the client type.
+func (k Keeper) GetAllClientIDsOfType(ctx sdk.Context, clientType string) (clientIDs []string) {
```

Comment:
This will be used in a followup pr

Reply:
I have a preference for not using named return values e.g. `(clientIDs []string)`. I think it's more readable to be more explicit with and just declare the slice next to its usage

i.e.

```golang
        var clientIDs []string
	for ; iterator.Valid(); iterator.Next() {
                ...
		clientIDs = append(clientIDs, clientID)
	}
```

WDYT?